memgpt/agent_store/db.py

```diff
@@ -192,6 +193,7 @@ def to_record(self):
                     role=self.role,
                     name=self.name,
                     text=self.text,
```

Comment:
Similarly, should delete/deprecate `text` and replace with `content` which can be `None`, `str` (text-only), or `List[dict/MultiModalMessagePart]` (multi-modal).